x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/step_configure_package.tsx

```diff
@@ -98,14 +110,22 @@ export const StepConfigurePackagePolicy: React.FunctionComponent<{
           })}
         </EuiFlexGroup>
       </>
-    ) : (
-      <CustomPackagePolicy
-        from={from}
-        packageName={packageInfo.name}
-        packagePolicy={packagePolicy}
-        packagePolicyId={packagePolicyId}
+    ) : !hasUiExtension ? (
```

Comment:
Mind flipping this to avoid the double negative? 

hasUiExtension ? X : Y

Reply:
Good point. I made the change. Thanks.